heartbeat/CTDB.in

```diff
@@ -243,7 +260,10 @@ Full path to the CTDB cluster daemon binary.
 <shortdesc lang="en">CTDB Daemon binary path</shortdesc>
 <content type="string" default="/usr/sbin/ctdbd" />
 </parameter>
+END
 
+	if [ -z "$fourdotnine" ]; then
```

Comment:
It wont know if it's >= 4.9 when showing metadata, so you can remove these if's, and add a notes like e.g. "Only affects 4.9+" or similar based on which versions use the different parameters.